lib/blueprinter/view_collection.rb

```diff
@@ -29,8 +29,10 @@ def fields_for(view_name)
       (identifier_fields + sorted_fields).reject { |field| excluded_fields.include?(field.name) }
     end
 
-    def transformers(view_name)
-      views[view_name].transformers
+    def transformers_for(view_name)
+      view_transformers = Set.new(transformers(view_name))
+      views[view_name].included_view_names.each { |included_view_name| view_transformers |= transformers(included_view_name) }
```

Comment:
This only works for directly included views, but not nested included views.
```
view :normal do
  transform TestTransformer
  fields :first_name
end

view :extended do
  include_view :normal
end

view :all do
  include_view :extended
end
```

Reply:
Done